src/ObjLoading/Game/T6/BSP/BSPUtil.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include "BSP.h"
+
+namespace BSP
+{
+    class BSPUtil
+    {
+    public:
+        static std::string getFileNameForBSPAsset(std::string& assetName);
```

Comment:
nit: Methods are generally named PascalCase in the codebase

adjusted in: 3d1fa7c6d9d80366143938b0f1910f71033066c2